emData/download.sh

```diff
@@ -3,17 +3,17 @@ set -e
 
 #### fw_synch_231205 ####
```

Comment:
@aryd Could you please replace `fw_synch_231205`, both here and in README.md, with the commit you used to generate these test vectors, even if it only exists in a development branch of cms-L1TK/cmssw for now?

Reply:
Updated comment in download.sh. But not sure what the command is to checkout this commit to add in the README.md file. I added this as a comment for now.

Reply:
Updated to specify the tag used in CMSSW to generate the test vectors.